app/assets/v2/js/board.js

```diff
@@ -195,7 +195,7 @@ Vue.mixin({
         vm.fetchContributorBounties('work_submitted');
         vm.fetchContributorBounties('interested');
       }
-      if (!Object.keys(vm.pTokens).length && persona === 'personal-tokens') {
+      if (persona === 'personal-tokens') {
         vm.fetchTokens('request');
         vm.fetchTokens('accepted');
         vm.fetchTokens('completed');
```

Comment:
We should now also add `vm.fetchTokens('cancelled');` here, right?

Reply:
:ok_hand: